scripts/helpers/Deployments.sol

```diff
@@ -103,6 +103,7 @@ abstract contract Deployments {
 
         // admin control
         address pauser;
+        address unpauser;
```

Comment:
note that there could be multiple wallets with these roles

Reply:
yeah, I'm going to refactor this logic to take multiple addresses for both pauser and unpauser in a different PR.